cli/index_data.py

```diff
@@ -46,7 +46,10 @@ def get_metadata_dict(task: IndexerInput) -> dict:
     :return dict: key-value pairs for metadata fields
     """
 
-    task_dict = {**task.dict(), **task.document_metadata.dict()}
+    task_dict = {
+        **task.dict(),
```

Comment:
I'm interested - do we really need the metadata in this dict twice? or can we remove the top-level `metadata` key now they're expanded into top-level entries?

Reply:
@diversemix good catch - it's actually filtered out in the next line as `document_metadata` isn't in `src.index_mapping.COMMON_FIELDS` but I've removed it from the top level as it seems more sensible to remove the duplication for any extra work we might do on `task_dict` in the future